test/unit-tests/action.mlt

```diff
@@ -13,18 +13,26 @@ let infer (a : Action.t) =
   (List.map (Path.Set.to_list x.deps) ~f:Path.to_string,
    List.map (Path.Set.to_list x.targets) ~f:Path.to_string)
 [%%ignore]
+File "test/unit-tests/action.mlt", line 7, characters 5-25:
```

Comment:
Commenting on this test so that we don't forget to change it

Reply:
Fixed this.